### DISCUSSION TITLE
Can't activate copilot (on a new computer)

### DISCUSSION THREAD

**Discussion by veraAlan** [community/community#13527] [Projects and Issues]
Been using copilot for a couple months now and I've created a similar workspace on my laptop. Since I've been using this tool on another computer, I thought that the authentication process would be pretty straight forward once I log into github. That was not the case and the text (screenshot below) shows up. I tried relogging and even reinstalling everything with no avail. 
Should I get in waitlist (again) just to use it in my laptop?
![image](https://user-images.githubusercontent.com/93807967/160189431-1220aee0-0e27-4871-8ba8-232fd8e8eb9b.png)

**Comment by MartinCura** [comment]
Same exact problem here, changed computer and says i can't use Copilot.

Is there a way to get the PAT from my previous computer so i try with the same one?

![image](https://user-images.githubusercontent.com/14017665/160190313-a5471f94-e830-424d-9315-ffb3fc849ddd.png)

Now it says "Copilot activation failed" even if i remove and reinstall the extension. Weirdly in the log it says the token is missing:

```logs
[INFO] [auth] [2022-03-25T19:43:03.013Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-03-25T19:43:03.014Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```

**Reply by MartinCura** [reply]
According to this the error is consistent with your account not being enabled, which isn't true. https://github.com/github/feedback/discussions/10474#discussioncomment-2030732

Would love to get this fixed as am very passionate about Copilot ! </3
